src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs

```diff
@@ -246,7 +251,7 @@ private async Task<string> GetLatestMatchingBundleVersionAsync(HttpClient httpCl
             return matchingBundleVersion;
         }
 
-        private static string FindBestVersionMatch(VersionRange versionRange, IEnumerable<string> versions)
+        internal static string FindBestVersionMatch(VersionRange versionRange, IEnumerable<string> versions, string bundleId, FunctionsHostingConfigOptions configOption)
```

Comment:
Same thing here, I feel like you should be able to just reference `configOption` without making this change - if I'm missing some context please let me know :)

Reply:
can't access configOption directly since FindBestVersionMatch a static method